docs/src/main/docs/tracing/01_tracing.adoc

```diff
@@ -57,3 +57,72 @@ ServerConfiguration.builder()
 ----
 <1> The name of the application (service) to associate with the tracing events
 <2> The endpoint for tracing events, specific to the tracer used, usually loaded from Config
+
+=== Configuring Tracing with Helidon MP
+There is an additional dependency that adds Tracing support to MP server and Jersey client:
+
+[source,xml,subs="verbatim,attributes"]
+.MP Tracing
+----
+<dependency>
+    <groupId>io.helidon.microprofile.tracing</groupId>
+    <artifactId>helidon-microprofile-tracing</artifactId>
+</dependency>
+----
+
+The minimal required configuration is `tracing.service` that contains
+the service name to be associated with tracing spans sent by this instance.
+
+All tracer specific configuration is expected in config under `tracing`.
+
+Example `microprofile-config.properties` with tracer configuration:
+
+----
+tracing.service=helidon-mp
+tracing.protocol=https
+tracing.host=tracer.example.com
+tracing.port=9411
+----
+
+== Trace propagation across services
+Tracing propagation is supported currently only with Jersey client.
+
+Tracing propagation works automatically in Helidon MP as long as you execute
+the client call in the same thread as the Jersey server side.
+
+_Exceptions that require manual handling_:
+
+- When the resource method is annotated with Fault Tolerance annotations (e.g. `@Fallback`)
+- When you use `async()` on the Jersey client request
+- When you are not in a Jersey Server scope (e.g. when using Jersey client from Helidon SE)
+
+In such cases, you must provide the SpanContext by hand:
+[source,java]
+.Tracing propagation with Jersey client (on a different thread)
+----
+import static io.helidon.tracing.jersey.client.ClientTracingFilter.CURRENT_SPAN_CONTEXT_PROPERTY_NAME;
+import static io.helidon.tracing.jersey.client.ClientTracingFilter.TRACER_PROPERTY_NAME;
+
+...
```

Comment:
Please use `//...`, it's better for syntax highlighting.

Reply:
Fixed